internal/aws/kinesis/kinesis.go

```diff
@@ -283,3 +283,33 @@ func (a *API) UpdateShards(ctx aws.Context, stream string, shards int64) error {
 
 	return nil
 }
+
+// GetTags recursively retrieves all tags for a Kinesis stream.
+func (a *API) GetTags(ctx aws.Context, stream string) ([]*kinesis.Tag, error) {
+	var tags []*kinesis.Tag
+	var lastTag string
+
+	more := true
+	for more {
```

Comment:
nit: might be slightly more idiomatic to remove the `more` boolean and write this as an infinite loop that would `break` when `resp.HasMoreTags` is false

Reply:
fixed 👍